### ISSUE TITLE
MissingMethodException on running debugger in vs2019

### ISSUE THREAD

**Issue by digittigerw** [microsoft/MIEngine#994]
Hi, 

I ported my extension from vs2017 to vs2019.  When I start my debugger based on MIEngine, I ran into this error:
System.MissingMethodException: 'Method not found: 'Microsoft.VisualStudio.ProjectSystem.IConfiguredProjectServices Microsoft.VisualStudio.ProjectSystem.ConfiguredProject.get_Services()'.'

I have vs2019 v16.5.2,  and Microsoft.VisualStudio.ProjectSystem v15.8.243  (one installed from NuGet). And all my projects including MICore, MIDebugEngine etc are targeting .net framework 4.6.2.

I searched on the web, found some posts about the issue, but non of the solutions applies/works.  Could you help ? Thanks !

**Comment by pieandcakes** [comment]
If it helps, here is an [issue](https://github.com/Microsoft/dotnet-apiport/issues/738) and a [PR](https://github.com/microsoft/dotnet-apiport/pull/739/files) of someone else solving the same error when updating to VS2019. It might shed some light on what needs to happen within your extension.

**Comment by digittigerw** [comment]
@pieandcakes Yeah, you are right. First it's just my extension that access the project properties for starting debugger. Second, updating that ProjectSystem package to v16.0  solves the issue.  Thanks !
